### DISCUSSION TITLE
Why is my template not working?

### DISCUSSION THREAD

**Discussion by rubenpulido1** [community/community#40665] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I tried using this template for my portfolio: https://github.com/manuelernestog/astro-modern-personal-website 

However my site is only displaying Hello World (https://rubenpulido1.github.io), why is it not showing the site as shown on:
https://astro-modern-personal-website.netlify.app/

<img width="900" alt="Screen Shot 2022-12-02 at 6 48 58 PM" src="https://user-images.githubusercontent.com/77760285/205418929-d6a670a3-2e49-42b0-ba0d-11d2f84a253b.png">

Edit: I think I forgot that I need to "Deploy" the site.  I'll try that and hope it works now.

**Comment by DSDmark** [comment, marked as answer]
Hi @rubenpulido1,

Don't create multiple discussion like this. One discussion is enough. (https://github.com/community/community/discussions/40590) 

And also remember this. Before creating a new discussion, you need to close the previous one.

**Reply by rubenpulido1** [reply]
Ok I tried to close it but I do not see anywhere I can delete the discussion.

**Reply by DSDmark** [reply]
@rubenpulido1, 

Yes, that is the main reason. We can't able to delete discussion yet. Even if it doesn't make any sense. So, try to create discussion which is useful to others.
